docker-compose.yml

```
@@ -39,35 +39,61 @@ services:
     ports:
       - "3500:3500"
     volumes:
-      - .:/app
+      - ./server:/app
     working_dir: /app
-    entrypoint: sh -c "sleep 10 && npm install && npm run start:dev"
+    entrypoint: sh -c "sleep 10 && npm install -g pnpm && pnpm install && pnpm run start:dev"
     networks:
       - my-network
 
   keycloak:
     image: quay.io/keycloak/keycloak:latest
     container_name: keycloak
     environment:
-      DB_VENDOR: h2
-      KEYCLOAK_AUTH_SERVER_URL: http://localhost:8082
-      KEYCLOAK_REALM: referentiel-applications
-      KEYCLOAK_CLIENT_ID: referentiel-applications-api
-      KEYCLOAK_CLIENT_SECRET: HbOh6jTkUiOLMB8H8KUZs048KMRQ7y9g
+      DB_VENDOR: postgres
+      DB_ADDR: postgres
+      DB_DATABASE: keycloak
+      DB_USER: myuser
+      DB_PASSWORD: mypassword
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
       KEYCLOAK_IMPORT: /realm.json
+      # Fixer la clé secrète
+      KEYCLOAK_CLIENT_SECRET: xrSK3UlEQcHhqmUXyhlKMKg0fEQPfQBI
     ports:
       - "8082:8080"
     volumes:
       - ./temp/realm.json:/realm.json
```

```suggestion
      - ./realm.json:/realm.json
```
le fichier realm.json est a la racine